app/src/main/java/org/openimis/imispolicies/ClientAndroidInterface.java

```diff
@@ -143,6 +149,8 @@
     private General general = new General(AppInformation.DomainInfo.getDomain());
     private ArrayList<String> mylist = new ArrayList<String>();
 
+    private final String defaultRarPassword = ")(#$1HsD";
```

Comment:
Can we remove the default password and require the password with the settings screen before uncompressing the master data file?

Reply:
I wanted to say to remove the default password but I think this is a good way to simplify the work for countries which do not share their country specific source code. Even if they share the source code, the production version can have different default password. 
However, we have a configuration class https://github.com/openimis/policies_android_app_java/blob/master/app/src/main/java/tz/co/exact/imis/AppInformation.java where the implementation changes the server URL. I suggest to move the default RAR password here to have all configuration in same place. 
Thanks.

Reply:
Thank you for suggestions.
Before uncompressing the master data, user is requested for a password to MasterData.rar file (according to our previosuly assumptions).
Earlier, all zip/unzip methods used hard-coded password to zip/unzip any .rar file. Now, all zip/unzip methods use the rar password set by user or if the password has not been set use the default password.
Ok, I will move default RAR password to this configuration class.